app/models/Forms.py

```diff
@@ -15,5 +15,7 @@ class Forms (baseModel):
   website           = TextField()
   gallery           = ForeignKeyField(Galleries)
   cv                = ForeignKeyField(Files, related_name="cv_file")
-  personal_statement = ForeignKeyField(Files, related_name="personal_statment")
+  personal_statement= ForeignKeyField(Files, related_name="personal_statment")
+  submit_date       = DateTimeField()
```

Comment:
If you add new entires into the model, please update the dummy data file to match